src/models/expense.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const { sequelize } = require('../db.js');
+const { DataTypes } = require('sequelize');
+
+const Expenses = sequelize.define('Expenses', {
+  id: {
+    type: DataTypes.UUID,
+    defaultValue: DataTypes.UUIDV4,
+    primaryKey: true,
+  },
+  userId: {
+    type: DataTypes.UUIDV4,
+    primaryKey: true,
+  },
+  spentAt: {
+    type: DataTypes.DATE,
+    allowNull: false,
+  },
+  title: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  amount: {
+    type: DataTypes.NUMBER,
+    allowNull: false,
+  },
+  category: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  note: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  createdAt: {
+    type: DataTypes.DATE,
+    field: 'created_at',
+    allowNull: false,
+    defaultValue: DataTypes.NOW,
+  },
+}, {
+  tableName: 'expenses',
+  updatedAt: false,
+});
+
```

Comment:
Consider implementing relations

Reply:
I implemented this but for some reason it is not displayed here as "Outdated"